python/main.py

```diff
@@ -1,32 +1,170 @@
 import os
 import logging
 import pathlib
+import json
+import sqlite3
+import hashlib
+from sqlite3 import Error
+
 from fastapi import FastAPI, Form, HTTPException
 from fastapi.responses import FileResponse
 from fastapi.middleware.cors import CORSMiddleware
 
+DB_PATH = './mercari.sqlite3'
+
+
 app = FastAPI()
 logger = logging.getLogger("uvicorn")
 logger.level = logging.INFO
 images = pathlib.Path(__file__).parent.resolve() / "image"
-origins = [ os.environ.get('FRONT_URL', 'http://localhost:3000') ]
+origins = [os.environ.get('FRONT_URL', 'http://localhost:3000')]
+
 app.add_middleware(
     CORSMiddleware,
     allow_origins=origins,
     allow_credentials=False,
-    allow_methods=["GET","POST","PUT","DELETE"],
+    allow_methods=["GET", "POST", "PUT", "DELETE"],
     allow_headers=["*"],
 )
 
+# loading inventory
+with open('inventory.json') as js:
+    iv = json.load(js)
+
+# connect to db
+conn = sqlite3.connect(DB_PATH)
+c = conn.cursor()
+
+# create table
+c.execute("""CREATE TABLE IF NOT EXISTS items (
+                id INTEGER PRIMARY KEY,
+                name STRING,
+                category STRING,
+                image STRING
+          )""")
+
+# inv = [('jacket', 'fashion'), ('cat-cage', 'pet'), ('switch', 'entertainment')]
```

Comment:
usually it's considered good practice to delete unused code

https://emanuilslavov.com/remove-unused-code/

1. it doesn't add any value to the collaboration
2. it creates confusion
3. don't worry about history, if important part of the code is deleted, we can retrieve from source code history.

Reply:
Get it! Next time I will delete all unnecessary code before commit ;D